fhir-server/src/test/java/au/csiro/pathling/fhirpath/function/memberof/MemberOfFunctionTest.java

```diff
@@ -134,19 +124,19 @@ public void memberOfCoding() {
         Collections.singletonList(argumentExpression));
 
     // Invoke the function.
-    final FhirPath result = new MemberOfFunction(mockCodeMapper).invoke(memberOfInput);
+    final FhirPath result = new MemberOfFunction().invoke(memberOfInput);
 
     // The outcome is somehow random with regard to the sequence passed to MemberOfMapperAnswerer.
     final Dataset<Row> expectedResult = new DatasetBuilder()
         .withIdColumn()
         .withEidColumn()
         .withColumn(DataTypes.BooleanType)
-        .withRow("encounter-1", makeEid(0), false)
-        .withRow("encounter-1", makeEid(1), true)
+        .withRow("encounter-1", makeEid(0), true)
+        .withRow("encounter-1", makeEid(1), false)
         .withRow("encounter-2", makeEid(0), true)
-        .withRow("encounter-3", makeEid(0), true)
+        .withRow("encounter-3", makeEid(0), false)
         .withRow("encounter-4", makeEid(0), false)
-        .withRow("encounter-5", makeEid(0), false)
+        .withRow("encounter-5", makeEid(0), true)
         .withRow("encounter-6", null, null)
         .build();
```

Comment:
I've been trying to reconcile the differences in these expectations as compared to what is on v3. As far as I can tell the expectations on v3 are wrong, but these are correct...

Reply:
The expectations in v3 were based on the true/false values returned by the mock so they were probably OK as well. 
The problem with them was though that order of the mock values was different of the order in which they were assigned in the result dataset. 
These now reflect the actual belonging of the ValueSet returned by the mock terminologyServer.